DGRunkeeperSwitch/DGRunkeeperSwitch.swift

```diff
@@ -64,7 +64,18 @@ open class DGRunkeeperSwitch: UIControl {
         set { selectedBackgroundView.backgroundColor = newValue }
         get { return selectedBackgroundView.backgroundColor }
     }
-    
+	
+	@IBInspectable
+	// Set a custom corner radius or use the default
+	open var customCornerRadius: CGFloat? {
```

Comment:
What if a user wants `cornerRadius` to be always rounded as it was previously and be able to resize the switch control? 

The easiest way is to introduce new property `open var rounded: Bool = false`.

If you do not want to create new var, then I suspect that you have to override `bounds` and make calculations. An example:
```
override var bounds: CGRect {
    didSet {
        if cornerRadius == oldValue.height / 2.0 {
            cornerRadius = bounds.height / 2.0
        }
    }
}
```